when using goteo's vagrantfile, npm install fails

When I use master's branch, I ran into this error

```
vagrant@ubuntu-14:~/goteo$ npm install
npm WARN engine imagemin@3.2.2: wanted: {"node":">=0.10.0","npm":">=2.1.5"} (current: {"node":"0.10.37","npm":"1.4.28"})
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.37","npm":"1.4.28"})
npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.37","npm":"1.4.28"})
npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.37","npm":"1.4.28"})
npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.37","npm":"1.4.28"})
npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.37","npm":"1.4.28"})
npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.37","npm":"1.4.28"})
npm WARN deprecated minimatch@1.0.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN optional dep failed, continuing imagemin-svgo@4.2.1
npm WARN optional dep failed, continuing imagemin-gifsicle@4.2.0
npm WARN optional dep failed, continuing imagemin-jpegtran@4.3.2
npm WARN optional dep failed, continuing imagemin-optipng@4.3.0
npm ERR! Error: UNKNOWN, symlink '../rimraf/bin.js'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Linux 3.13.0-24-generic
npm ERR! command "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! cwd /home/vagrant/goteo
npm ERR! node -v v0.10.37
npm ERR! npm -v 1.4.28
npm ERR! path ../rimraf/bin.js
npm ERR! code UNKNOWN
npm ERR! errno -1
npm ERR! not ok code 0
```

in ububuntu 14.04, node is still in 0.10.x and npm < 2.x

If I use https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions, all green

--- a/vagrant.sh
+++ b/vagrant.sh
@@ -33,6 +33,7 @@ iptables-save
 service mysql restart
 
 # NPM
+curl -sL https://deb.nodesource.com/setup_6.x | bash -
 apt-get install nodejs -y
 npm install -g grunt-cli
 

